src/index.js

```diff
@@ -56,6 +56,10 @@ export default function (options = {}, callback = function noop () {}) {
 
     if (specFiles.length) {
       protractorArgs = protractorArgs.filter((arg) => !/^--suite=/.test(arg))
+      var specIndex = protractorArgs.indexOf('--specs')
```

Comment:
This should probably be a `let` just to be consistent (there's another `var` in this file that I will clean up).